pyproject.toml

```diff
@@ -0,0 +1,134 @@
+[build-system]
+requires = ["hatchling", "hatch-requirements-txt"]
+build-backend = "hatchling.build"
+
+[project]
+name = "amazon-braket-examples"
+version = "0.0.1"
+description = 'Example notebooks that show how to apply quantum computing with Amazon Braket.'
+readme = "README.md"
+requires-python = ">=3.9"
+license = "apache-2.0"
+keywords = []
+authors = [{name = "Amazon Web Services"},]
+classifiers = [
+  "Development Status :: 5 - Production/Stable",
+  "Intended Audience :: Developers",
+  "Intended Audience :: Education",
+  "Intended Audience :: Science/Research",
+  "Natural Language :: English",
+  "License :: OSI Approved :: Apache Software License",
+  "Programming Language :: Python",
+  "Programming Language :: Python",
+  "Programming Language :: Python :: 3.9",
+  "Programming Language :: Python :: 3.10",
+  "Programming Language :: Python :: 3.11",
+]
+dynamic = ["dependencies"]
+
+[tool.hatch.metadata.hooks.requirements_txt]
+files = ["requirements.txt"]
+
+[project.urls]
+Documentation = "https://github.com/unknown/amazon-braket-examples#readme"
+Issues = "https://github.com/unknown/amazon-braket-examples/issues"
+Source = "https://github.com/unknown/amazon-braket-examples"
+
+[tool.hatch.envs.default]
+dependencies = [
+  "jupyter",
+  "nbconvert",
+  "pytest",
+  "pytest-rerunfailures",
+  "pytest-xdist",
+  "testbook",
+]
+[tool.hatch.envs.default.scripts]
+test = "pytest -n auto -ra -v --durations=0 test/integ_tests/"
+
+
+[[tool.hatch.envs.all.matrix]]
+python = ["3.9", "3.10", "3.11"]
+
+[tool.hatch.envs.lint]
+detached = true
+dependencies = [
+  "black>=23.1.0",
+  "mypy>=1.0.0",
+  "ruff>=0.0.243",
+]
+[tool.hatch.envs.lint.scripts]
+typing = "mypy --install-types --non-interactive {args:examples tests}"
+style = [
+  "ruff {args:.}",
+  "black --check --diff {args:.}",
+]
+fmt = [
+  "black {args:.}",
+  "ruff --fix {args:.}",
+  "style",
+]
+all = [
+  "style",
+  "typing",
+]
+
+[tool.black]
+target-version = ["py39"]
+line-length = 120
+skip-string-normalization = true
+
+[tool.ruff]
+target-version = "py39"
+line-length = 120
+select = [
```

Comment:
Where did you get this list from? Again, not sure where/how this applies.

Reply:
This was from `hatch --init`. I have left this as a linting starting point. I can remove to avoid confusion.